packages/jsii-dotnet-generator/src/Amazon.JSII.Generator/Interface/InterfaceMethodGenerator.cs

```diff
@@ -11,9 +11,9 @@ public class InterfaceMethodGenerator : MethodGeneratorBase<InterfaceType>
         public InterfaceMethodGenerator(InterfaceType type, Method method, ISymbolMap symbols, INamespaceSet namespaces)
             : base(type, method, symbols, namespaces)
         {
-            if (method.IsAbstract)
+            if (!method.IsAbstract)
```

Comment:
Looking at `jsii-spec/lib/spec.ts`, `Method.isAbstract` was changed from made nullable in [this PR](https://github.com/awslabs/jsii/pull/90/files), which slipped my radar. So we need to change the following:

### Amazon.JSII.JsonModel

* Change the type of `Method.IsAbstract` from `bool` to `bool?`.
* Change the attribute of `Method.IsAbtract` from `[JsonProperty("abstract")]` to `[JsonProperty("abstract", NullValueHandling = NullValueHandling.Ignore)]`.
* Change the type of the `Method` constructor parameter `isAbstract` from `bool` to `bool?`, and give it a default value of null.

### Amazon.JSII.Generator

* Change this line from `if (!method.IsAbstract)` to `if (method.IsAbstract != true)`.

Reply:
✔️